test/cypress/integration/group-member-removal.spec.js

```diff
@@ -17,8 +17,11 @@ describe('Group - Removing a member', () => {
     cy.getByDT('groupMembers').find(`ul>li:nth-child(${nth})`).within(() => {
       cy.getByDT('username').should('contain', `${username}-${userId}`)
 
-      cy.getByDT('menuActions').find('button').click()
-      cy.getByDT('menuContent').find('ul>li:nth-child(2) button')
+      cy.getByDT('openMemberProfileCard').click()
+    })
+
+    cy.getByDT('memberProfileCard').within(() => {
+      cy.getByDT('buttonRemoveMember')
```

Comment:
A quick tip when using `data-test`. You can write just `removeMember` and use `button` as the second parameter to make sure it's indeed a `<button>`. `cy.getByDT('removeMember', 'button')`